.travis.yml

```diff
@@ -1,13 +1,38 @@
 sudo: false
+
 language: python
+
 cache: pip 
+
 python:
   - 3.5
+
+env:
+  - PYDATA=true
```

Comment:
PyData app introduces a migration since it adds a `choice` argument to the sponsorship amount field. So, I do not check for missing migrations here.